.github/workflows/package-unit-tests.yml

```diff
@@ -34,6 +34,11 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
+      # ninja-build is used by default if available and results in faster build times
+      - name: Install ninja
+        run: sudo apt-get install ninja-build
```

Comment:
Do we not need ccache? I saw it was installed in the other script.

Reply:
It was my assumption that it wasn't needed and the [ccache-action](https://github.com/hendrikmuhs/ccache-action) would bring it, which it does seem to do.